src/main/java/io/codelex/dateandtime/practice/DatePeriod.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.dateandtime.practice;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+
+public class DatePeriod {
+    public static void main(String[] args) {
+
+        LocalDate firstStart = LocalDate.of(2022, 1, 5);
+        LocalDate firstEnd = LocalDate.of(2022, 1, 7);
+
+        LocalDate secondStart = LocalDate.of(2022, 1, 2);
+        LocalDate secondEnd = LocalDate.of(2022, 1, 10);
+
+        intersection(firstStart, firstEnd, secondStart, secondEnd);
+    }
+
+    public static void intersection(LocalDate firstStart, LocalDate firstEnd, LocalDate secondStart, LocalDate secondEnd) {
```

Comment:
`The intersection method should receive DatePeriod as a parameter and return DatePeriod - the intersection of both periods.`

Reply:
unfortunately it be like that sometimes, happens to everyone 😄

Reply:
šis jau bija tas kurš pietrūka :smile: